src/apps/kwenta/optimism/kwenta.staking-v2.contract-position-fetcher.ts

```diff
@@ -0,0 +1,67 @@
+import { Inject } from '@nestjs/common';
+
+import { APP_TOOLKIT, IAppToolkit } from '~app-toolkit/app-toolkit.interface';
+import { PositionTemplate } from '~app-toolkit/decorators/position-template.decorator';
+import { getLabelFromToken } from '~app-toolkit/helpers/presentation/image.present';
+import { isSupplied } from '~position/position.utils';
+import {
+  GetDataPropsParams,
+  GetDisplayPropsParams,
+  GetTokenBalancesParams,
+} from '~position/template/contract-position.template.types';
+import {
+  SingleStakingFarmDataProps,
+  SingleStakingFarmDefinition,
+  SingleStakingFarmTemplateContractPositionFetcher,
+} from '~position/template/single-staking.template.contract-position-fetcher';
+
+import { KwentaContractFactory, KwentaStakingV2 } from '../contracts';
+
+const FARMS = [
+  {
+    address: '0x61294940ce7cd1bda10e349adc5b538b722ceb88',
+    stakedTokenAddress: '0x920cf626a271321c151d027030d5d08af699456b', // Kwenta
+    rewardTokenAddresses: ['0x920cf626a271321c151d027030d5d08af699456b'], // Kwenta
+  },
+];
+
+@PositionTemplate()
+export class OptimismKwentaStakingV2ContractPositionFetcher extends SingleStakingFarmTemplateContractPositionFetcher<KwentaStakingV2> {
+  groupLabel = 'Staking V2';
+
+  constructor(
+    @Inject(APP_TOOLKIT) protected readonly appToolkit: IAppToolkit,
+    @Inject(KwentaContractFactory) protected readonly contractFactory: KwentaContractFactory,
+  ) {
+    super(appToolkit);
+  }
+
+  getContract(address: string): KwentaStakingV2 {
+    return this.contractFactory.kwentaStakingV2({ address, network: this.network });
+  }
+
+  async getLabel({ contractPosition }: GetDisplayPropsParams<KwentaStakingV2>) {
+    const suppliedToken = contractPosition.tokens.find(isSupplied)!;
+    return `Staked ${getLabelFromToken(suppliedToken)}`;
```

Comment:
I think adding `Staked` to the label will be redundant. wdyt ?

Reply:
It's not redudant in the global portfolio view but it is in the app view, example for Thales staking:
![image](https://github.com/Zapper-fi/studio/assets/70271293/ca0a9d94-e572-408d-a2d2-8bf64306ebfc)

![image](https://github.com/Zapper-fi/studio/assets/70271293/d2edd3ed-47c8-4631-b1e9-3a809d05bc89)

I like it better this way personally

Reply:
Okok, so it won't be in the portfolio, but it will be on the app page under asset because all position will have a 'staked' prefix, but it's fine. I'll merge it ! Good job !